### PR TITLE
 turning on LLG as default in cmake

### PR DESCRIPTION
**Pull request by RevathiJambunathan**
to ensure that CI compiles with LLG=ON

### CODE REVIEW

**Review by jackieyao0114 — APPROVED** [review]

File: Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.H [hunk 1]
```diff
@@ -175,29 +175,30 @@ public:
...
+         }
+         else if(n==2){
+             amrex::Real a_cross_b_z = a(i, j, k, 0) * b(i, j, k, 1) -
+                                       a(i, j, k, 1) * b(i, j, k, 0);
+             M_field = ( b(i, j, k, 2) + a_dot_b * a(i, j, k, 2) - a_cross_b_z ) / ( 1.0 + a_square);
+         }
+         else{
+             printf("n=%d\n", n);
+             amrex::Abort("Wrong component n of the M_field");
+         };
```
Comment (RevathiJambunathan) [inline Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.H]:
```suggestion
         }
```

File: Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveEPML.cpp [hunk 2]
```diff
@@ -40,7 +40,11 @@ void FiniteDifferenceSolver::MacroscopicEvolveEPML (
    // Select algorithm (The choice of algorithm is a runtime option,
    // but we compile code for each algorithm, using templates)
 #ifdef WARPX_DIM_RZ
+#    ifndef WARPX_MAG_LLG
     amrex::ignore_unused(Efield, Bfield, Jfield, Ffield, sigba, dt, pml_has_particles, mu_mf, eps_mf, sigma_mf, macroscopic_properties);
+#    else
+    amrex::ignore_unused(Efield, Hfield, Jfield, Ffield, sigba, dt, pml_has_particles, mu_mf, eps_mf, sigma_mf, macroscopic_properties);
+#endif
```
Comment (RevathiJambunathan) [inline Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveEPML.cpp]:
```suggestion
#    endif
```

File: Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp [hunk 3]
```diff
@@ -225,8 +225,6 @@ void FiniteDifferenceSolver::MacroscopicEvolveHMCartesian(
                         // check the normalized error
                         if (amrex::Math::abs(1._rt - M_magnitude_normalized) > mag_normalized_error)
                         {
-                            printf("i = %d, j=%d, k=%d\n", i, j, k);
-                            printf("M_magnitude_normalized = %f, mag_normalized_error=%f\n", M_magnitude_normalized, mag_normalized_error);
```
Comment (RevathiJambunathan) [inline Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp:224]: removing print statements here as SYCL did not compile and threw an error

Comment (ajnonaka) [reply Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveHM.cpp:224]: I saw that; thank you!

**Review by ajnonaka — APPROVED** [review]
IntVect's look good!  I approve for merging.

**Pull request merged**